docs/contributor/03-11-hap-rules.md

```diff
@@ -0,0 +1,207 @@
+# HAP Rule Configuration
+
+> [!NOTE]
+> This feature is still being developed and will be available soon.
+
+ The Hyperscaler Account Pool (HAP) Rule Configuration is a functionality that allows to control pool selection from a configurable helm property. 
+ The HAP pool is a set of Gardener's SecretBindings K8S resources that share the same labels. 
+ The HAP Pool Selection is a process that for every skr creation request decides on which secret binding is the most suitable to use. 
+ The process generates a list of search labels based on skr's physical and or logical (calculated) attributes.
```

Comment:
In principle we generate search labels based on plan, provider, hyperscaler region, platform region.
I am not sure which attributes are physical and which are logical (calculated). 
If the only physical is hyperscaler region (as this is physical location of the cluster) and remaining are logical we could manage without this distinction physical vs logical. Or define it explicitly to make this more clear for the reader.

Reply:
Physical vs logical distinction removed.